docs/development/workflow/get_devel_version.rst

```diff
@@ -274,6 +274,9 @@ Testing is an important part of making sure astropy produces reliable,
 reproducible results. Before you try out a new feature or think you have found
 a bug make sure the tests run properly on your system.
 
+before runnig tests, see
+:ref:`testing-dependencies`.
```

Comment:
```suggestion
Before running your tests, please see :ref:`testing-dependencies`.
```

Reply:
Thanks for pointing it out. next time I will always be double check for grammatical error or typo.

Reply:
Thanks for pointing it out. Next time I will always be double check for grammatical error or typo
.